flask_admin/translations/af/LC_MESSAGES/admin.po

```diff
@@ -594,3 +621,6 @@ msgstr "Kies ten minste een rekord."
 
 #~ msgid "File \"%s\" already exists."
 #~ msgstr "Lêer \"%s\" bestaan reeds."
+
+#~ msgid "Export type \"%(type)s\" not supported."
+#~ msgstr "Uitvoertipe \"%(type)s\" word nie ondersteun nie."
```

Comment:
Backup comments like this one still aren't necessary. (They're meant to be temporary changes for cases like what happened here – when the English string is changed slightly; rather than being committed.)